src/rpc.cpp

```diff
@@ -133,6 +134,27 @@ RPCServer::RPCServer() {
         RPCObjectID realm_id = store_object(realm_object);
         return (json){{"result", realm_id}};
     };
+    m_requests["/create_user"] = [this](const json dict) {
```

Comment:
Why not just forward these requests to the Sync object as we do with all other methods? Same question for the rest of new sync methods.